mappings/net/minecraft/entity/mob/ZombieVillagerEntity.mapping

```diff
@@ -11,4 +11,5 @@ CLASS aut net/minecraft/entity/mob/ZombieVillagerEntity
 	METHOD a setConverting (Ljava/util/UUID;I)V
 	METHOD a finishConversion (Lve;)V
 	METHOD dX isConverting ()Z
+	METHOD ea getConversionRate ()I
```

Comment:
What's "conversion rate"? Do you mean conversion *speed* or conversion *progress*?

Reply:
No. It is a rate for one tick, that in that tick, at 1/100 chance, the conversion will check for beds and iron bars around and return a number higher than one to reduce the ticks it take to convert the zombie villager.